src/Aspire.Hosting/Dcp/Model/Container.cs

```diff
@@ -178,6 +178,10 @@ internal sealed class ContainerStatus : V1Status
     [JsonPropertyName("effectiveEnv")]
     public List<EnvVar>? EffectiveEnv { get; set; }
 
+    // Effective values of launch arguments to be passed to the Container, after all substitutions are applied.
+    [JsonPropertyName("effectiveArgs")]
+    public List<string>? EffectiveArgs { get; set; }
```

Comment:
I assume this means all these effective args were already being passed over the wire, we just weren't deserializing them (and thus not making use of them).

Reply:
Correct.